### DISCUSSION TITLE
CUDA version on your system does not match the CUDA version torch was compiled with.

### DISCUSSION THREAD

**Discussion by anash08** [NVIDIA-NeMo/NeMo#7089] [Q&A]
While installing Apex, it may raise an error if the CUDA version on your system does not match the CUDA version torch was compiled with. This raise can be avoided by commenting it here:
if (bare_metal_version != torch_binary_version):
        raise RuntimeError(
            "Cuda extensions are being compiled with a version of Cuda that does "
            "not match the version used to compile Pytorch binaries.  "
            "Pytorch binaries were compiled with Cuda {}.\n".format(torch.version.cuda)
            + "In some cases, a minor-version mismatch will not cause later errors:  "
            "https://github.com/NVIDIA/apex/pull/323#discussion_r287021798.  "
            "You can try commenting out this check (at your own risk)."
        )
        by default it commented 
        iam trying to run this on AWS sage maker using ml.g5.12xlarge
        unable to get through Please help me with this.
        Using python 3.8.10 but the documentation says use Python 3.9 or above is this a typo or am i doing something worng please help.

**Comment by kiranshivaraju** [comment]
++ i am finding the same issue when i am trying to install nemo..on aws
